### ISSUE TITLE
Regex pattern that injects createObject("roSGNode", "ODC") can cause certain roku apps to crash 

### ISSUE THREAD

**Issue by AntonKostenko** [dlenroc/node-roku#75]
The regex pattern `const pattern = /(\.show\(\))/gim;` that injects `createObject("roSGNode", "ODC")` can cause unexpected injections in some scenarios.
I am currently working on a couple of Roku channels that set a string to `showEventMessage = "example.show()"`.

When using the current pattern, this injects an additional unexpected `createObject("roSGNode", "ODC")` that looks like this:
```
showEventMessage = "example.show() : createObject("roSGNode", "ODC")"
```

The misplaced quotes cause compilation errors.
Unfortunately I am unable to modify the roku app but a workaround I found was changing the pattern to: 
```
const pattern = /(\.show\(\))(?![\w"])/gim;
```
https://github.com/dlenroc/node-roku/blob/main/packages/odc/src/internal/injector.ts#L61

Would you consider publishing a new package that account for this scenario?

**Comment by dlenroc** [comment]
Thanks @AntonKostenko for finding and reporting this case.  

**PS:** A fix for this case was added via #76 and became part of version [1.4.3](https://www.npmjs.com/package/@dlenroc/roku-odc/v/1.4.3).

**Comment by AntonKostenko** [comment]
Thank you @dlenroc for the fix. Also thank you for this repo. You have created a very interesting and strong tool for such a difficult platform.
